src/main.rs

```diff
@@ -127,9 +133,15 @@ pub fn parse_args() -> MiniserveConfig {
                 .validator(is_valid_auth)
                 .help("Set authentication (username:password)")
                 .takes_value(true),
+        ).arg(
+            Arg::with_name("no-symlinks")
+                .short("P")
+                .long("no-symlinks")
+                .help("Do not follow symbolic symbolic links"),
```

Comment:
Double `symbolic`

Reply:
Sorry about that. fc46689 should fix that.